package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@salesforce/plugin-release-management",
   "description": "A plugin for preparing and publishing npm packages",
-  "version": "2.2.2",
+  "version": "2.2.4",
```

Comment:
package.json version is bumped to `2.2.4` because I accidentally released `2.2.3` under the `dev` tag for testing.